Sources/TuistDependencies/SwiftPackageManager/Utils/SwiftPackageManagerGraphGenerator.swift

```diff
@@ -86,9 +86,11 @@ public final class SwiftPackageManagerGraphGenerator: SwiftPackageManagerGraphGe
             let name = dependency.packageRef.name
             let packageFolder: AbsolutePath
             switch dependency.packageRef.kind {
-            case "remote":
+            case "remote" where workspaceState.version < 5,
+                 "remoteSourceControl" where workspaceState.version >= 5:
                 packageFolder = checkoutsFolder.appending(component: dependency.subpath)
```

Comment:
I would just ignore the version (same also below):
```suggestion
            case "remote", "remoteSourceControl":
                packageFolder = checkoutsFolder.appending(component: dependency.subpath)
```